vars/pipelineSnapshotTesting.groovy

```diff
@@ -125,37 +115,60 @@ void call(Map config=[:]) {
                                     """
                                 }
                             }
+                            'data node config': {
+                                withCredentials([sshDevnetCredentials]) {
+                                    sh label: "scp data node config from ${remoteServer}",
+                                        script: """#!/bin/bash -e
+                                            scp -i \"\${PSSH_KEYFILE}\" \"\${PSSH_USER}\"@\"${remoteServer}\":/home/vega/vega_home/config/data-node/config.toml data-node-config.toml
+                                        """
+                                    sh label: "print genesis.json", script: """#!/bin/bash -e
+                                        cat data-node-config.toml
```

Comment:
I do not know if it is a good idea just to print it, as there is a database password in text format.

Reply:
Yes, better not to print this config - there are two secrets in that config: DB password and DeHistory private key.
If you want to check if the file downloaded successfully, then maybe you can use `dasel` here to extract the `dehistor_peers`?